genai-perf/README.md

```diff
@@ -73,45 +73,34 @@ INSTALLATION
 
 ## Installation
 
-The easiest way to install GenAI-Perf is through
-[Triton Server SDK container](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver).
-Install the latest release using the following command:
+The easiest way to install GenAI-Perf is through pip.
+### Install Perf Analyzer (Ubuntu 24.04, Python 3.10+)
```

Comment:
I find some of this confusing. Do we need to talk about installing PA in the GAP instructions? If so, we should have sections at the same header level (e.g. H3 `###`) for each step in the install process. I think we could remove it though.

Reply:
It should be GenAI-Perf and not PA here. Thank you.